user.js

```diff
@@ -1225,7 +1225,7 @@ user_pref("mailnews.headers.useMinimalUserAgent", true);
  * [SETTING] Preferences > Composition > Addressing > Automatically add outgoing e-mail addresses...
  * [SETTING][CARDBOOK] CardBook > Preferences > Email > Collect Outgoing Email ***/
    // user_pref("mail.collect_addressbook", "jsaddrbook://history.sqlite");
-user_pref("mail.collect_email_address_outgoing", false);
+// user_pref("mail.collect_email_address_outgoing", false);
```

Comment:
Same : I find this feature very intrusive and store on disk (in an unencrypted way) the list of contacts you wrote to. I'd also vote for keeping it disabled and let overrides prevail for people wanted it.

Reply:
there is no updating mechanism for overrides to my knowledge. But okay.